addons/website_sale/models/product_template.py

```diff
@@ -287,14 +287,16 @@ def _get_sales_prices(self, website):
             if pricelist_rule_id:  # If a rule was applied, there might be a discount
                 # For ecommerce flows, the base price is always the product sales price
                 # which can be computed by calling `_compute_base_price` without a pricelist rule
+                pricelist_item = self.env['product.pricelist.item'].browse(pricelist_rule_id)
+                pricelist_compute_price = pricelist_item.compute_price
                 pricelist_base_price = template.env['product.pricelist.item']._compute_base_price(
                     product=template,
                     quantity=1.0,
                     date=date,
                     uom=template.uom_id,
                     currency=currency,
                 )
-                if pricelist_base_price != pricelist_price:
+                if pricelist_base_price != pricelist_price and pricelist_compute_price == "percentage":
```

Comment:
why we only want to display this for `percentage` price not for other `formula` and `fixed` price? :thinking:

Reply:
@dalaOdoo mentioned on the task and suggested that the strikethrough should only be `applied for discount` and `not for fixed or formula`

Reply:
The biggest problem we need to solve is the fact that currently a pricelist in another currency behave as a discount with the strike through price.